internal/keppel/account.go

```diff
@@ -30,10 +27,9 @@ type Account struct {
 	Name              models.AccountName    `json:"name"`
 	AuthTenantID      string                `json:"auth_tenant_id"`
 	GCPolicies        []GCPolicy            `json:"gc_policies,omitempty"`
-	InMaintenance     bool                  `json:"in_maintenance"`
-	Metadata          map[string]string     `json:"metadata"`
```

Comment:
When we discussed this, I didn't realize that these fields are not `omitempty`, so it's always shown on all accounts. Because of that, I would like to keep them around, always set to `false` and `nil` respectively, at least until we have removed usage of these fields from the Elektra UI.